CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Changelog - v3
 
+## [v3.4.8] (May 19 2023)
+
+Fixes:
+* Prevent white space only text sending
+* Mentioned user Regex parsing
+  Mention will now work even if userId has `.*+?^${}()|[\]\\` characters
+* ChannelList blink when when message is send
+  Happened when there were two channelLists in the same page with
+  different query params
```

Comment:
seems it doesn't need a line break?

Reply:
Is because I break lines at 80  chars(for description), it is a best practice,

Reply:
Ok ok